examples/cfd/07_cavity_flow.ipynb

```diff
@@ -0,0 +1,551 @@
+{
```

Comment:
<p>Should we execute for this error? Maybe avoid it?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>